src/schema/artwork/contextGrids/AuctionArtworkGrid.ts

```diff
@@ -0,0 +1,57 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import {
+  ContextGridType,
+  formDefaultGravityArgs,
+} from "schema/artwork/contextGrids"
+import { artworkConnection } from "schema/artwork"
+import { connectionFromArraySlice } from "graphql-relay"
+import { pageable } from "relay-cursor-paging"
+import { map } from "lodash"
+
+export const AuctionArtworkGridType = new GraphQLObjectType<
+  { sale: any; artwork: any },
+  any
+>({
+  name: "AuctionArtworkGrid",
+  interfaces: [ContextGridType],
+  fields: () => ({
```

Comment:
These all have to be thunks to avoid a circular dependency issue

Reply:
♻️